test/goHome.test.ts

```diff
@@ -0,0 +1,55 @@
+import { chromium, Page, Browser, BrowserContext } from "playwright"
+
+// NOTE: this is hard-coded and passed as an environment variable
+// See the test job in ci.yml
+const PASSWORD = "e45432jklfdsab"
```

Comment:
Maybe we should use `PASSWORD=e45432jklfdsab yarn test` in CI and then use `process.env.PASSWORD` here, so if a dev runs the test they can set the password used in their instance.

Reply:
Actually, even better, in the `ci.yaml` adding something like:

```
env:
  PASSWORD: e45432jklfdsab
```

Should set the env var for both the node spawn and the yarn test.

Reply:
Genius! I'll add that now.